product/roundhouse.console/Program.cs

```diff
@@ -429,6 +432,12 @@ private static void run_diff_utility(ConfigurationPropertyHolder configuration)
             }
         }
 
+        public static void init_security_protocol()
+        {
+            // allow tls
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12 | SecurityProtocolType.Ssl3;

```

Comment:
Nevemind - looks like @ChrisMissal already upgraded to .NET 4.5 awhile back - https://github.com/chucknorris/roundhouse/commit/b0990af067edd589b0eee4fd1432ad9016768f7a